doc/src/manual/mathematical-operations.md

```diff
@@ -348,16 +359,18 @@ Julia applies the following order of operations, from highest precedence to lowe
 | Control flow   | `&&` followed by `\|\|` followed by `?`                                                           |
 | Assignments    | `= += -= *= /= //= \= ^= ÷= %= \|= &= ⊻= <<= >>= >>>=`                                            |
 
-### Elementary Functions
+For a complete list of *every* Julia operator's precendence, see the top of this file:
+https://github.com/JuliaLang/julia/blob/master/src/julia-parser.scm
```

Comment:
It *is* supported, but I thought that since this case was a link to code, keeping the url explicit made sense. But I could also change it.

Do you have a suggestion? Maybe just something like this?:

```
[julia/src/julia-parser.scm](https://github.com/JuliaLang/julia/blob/master/src/julia-parser.scm)
```